tests/cli/demultiplex/test_demultiplex_flowcell.py

```diff
@@ -27,12 +28,17 @@ def test_demultiplex_flow_cell_dry_run(
     flow_cell: FlowCellDirectoryData = FlowCellDirectoryData(
         tmp_flow_cells_directory_ready_for_demultiplexing_bcl2fastq
     )
+    add_sample_sheet_path_to_housekeeper(
+        flow_cell_directory=tmp_flow_cells_directory_ready_for_demultiplexing_bcl2fastq,
+        flow_cell_name=flow_cell.id,
+        hk_api=demultiplexing_context_for_demux.housekeeper_api,
+    )

```

Comment:
For the future this should be a function is store helpers, like `ensure_sample_sheet`

Reply:
and all appearances below